plenum/test/node_catchup/helper.py

```diff
@@ -37,11 +37,21 @@ def chk_ledger_and_state(first_node, second_node, ledger_id):
         else:
             logger.debug("Excluding check_last_ordered_3pc check")
 
+        if exclude_from_check and 'check_view_no' not in exclude_from_check:
```

Comment:
`exclude_from_check ` is None by default, so it looks like `check_view_no`, `check_primaries_equality`, `check_last_ordered_3pc` and `check_seqno_db_equality` are never called?

Reply:
Was it fixed by your PR?